test/actions/katello/applicability/generate_applicability_test.rb

```diff
@@ -0,0 +1,29 @@
+require 'katello_test_helper'
+
+module ::Actions::Katello::Applicability::Host
+  class GenerateApplicabilityTest < ActiveSupport::TestCase
+    include Dynflow::Testing
+    include Support::Actions::Fixtures
+    include FactoryBot::Syntax::Methods
+
+    before :all do
+      User.current = users(:admin)
+      @host = FactoryBot.build(:host, :with_content, :with_subscription, :content_view => katello_content_views(:library_dev_view),
+                               :lifecycle_environment => katello_environments(:library))
+      SETTINGS[:katello][:katello_applicability] = true
```

Comment:
anytime we set this to true, we need to set it back to false in the test teardown, that should make it not affect other tests